WISE-Community/WISE-Client
# Show links to Discourse category and latest topics in LibraryProjectDetails

## Changes
If unit metadata contains a valid `discourseCategoryURL`, LibraryProjectDetails will show links to the category and the 3 latest topics, as well as the total number of posts in the category.

I also updated some styles.

Closes #163.

## Test
- Add a key-value pair to a unit's `metadata` in the project.json file (e.g. `"discourseCategoryURL": "https://wise-discuss.berkeley.edu/c/wise-curricula/what-makes-a-good-cancer-medicine/40"`) using the authoring tool (Advanced -> Show JSON).
- Find the unit on the site homepage or in your Unit Library (/teacher/home/library/*). Click on the unit to bring up the details dialog.
- Confirm that links to the Discourse category and the latest posts (up to 3) are added to the project details content and that the total number of posts is displayed in parentheses after "Discuss".
- Verify that no links are added and the details dialog displays correctly if the `discourseCategoryURL` in the project metadata is not a valid Discourse category.

![image](https://user-images.githubusercontent.com/297450/119700670-5af5b500-be08-11eb-93a3-61612449a625.png)
